Drivers/Periphs/Src/radio.c

```diff
@@ -61,7 +64,7 @@ ErrorStatus Ethernet_Init() {
     sLocalAddr.sin_addr.s_addr = htonl(lwip_makeu32_func(IP4_SERVER_ADDRESS));
     sLocalAddr.sin_port = htons(SERVER_PORT);
 
```

Comment:
initialization is missing a call to socket(), bind(), and listen():
https://pubs.opengroup.org/onlinepubs/007908799/xns/accept.html

server initialization is a bit different from client where the server has a sequence of initialization calls.

Reply:
added calls to lwip_socket(), bind(), and listen() from EthernetNew branch